sdk/attestation/azure-security-attestation/azure/security/attestation/_client.py

```diff
@@ -22,107 +17,151 @@
     InitTimeData,
     DataType,
     AttestSgxEnclaveRequest,
-    AttestOpenEnclaveRequest)
+    AttestOpenEnclaveRequest,
+)
 from ._configuration import AttestationClientConfiguration
-from ._models import (
-    AttestationSigner,
-    AttestationToken,
-    AttestationResponse,
-    AttestationResult,
-    AttestationData,
-    TpmAttestationRequest,
-    TpmAttestationResponse)
-import base64
-from azure.core.tracing.decorator import distributed_trace
-from threading import Lock
+from ._models import AttestationSigner, AttestationToken, AttestationResult
+from ._common import merge_validation_args
+
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from azure.core.credentials import TokenCredential
 
 
 class AttestationClient(object):
     """An AttestationClient object enables access to the Attestation family of APIs provided
```

Comment:
To eliminate the need to other suppression comments in this class (this can go here or after the docstring):
```suggestion
class AttestationClient(object):
    # pylint: disable = protected-access
    """An AttestationClient object enables access to the Attestation family of APIs provided
```